lib/datapacksjob.js

```diff
@@ -2951,6 +2951,8 @@ DataPacksJob.prototype.downloadPerformanceData = async function (jobInfo) {
 
         var i = 0;
 
+        var vloc=this.vlocity;
```

Comment:
Should just set `var self - this;` and use that below.

Reply:
done